sdks/python/pom.xml

```diff
@@ -56,6 +56,41 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>copy-project-files</id>
+              <phase>prepare-package</phase>
+              <goals>
+                <goal>run</goal>
+              </goals>
+              <configuration>
+                <tasks>
+                  <copy file="../../LICENSE" tofile="${basedir}/LICENSE"/>
```

Comment:
I wouldn't do this -- reaching outside the current context generally breaks things.